File: Benchmarks.FSharp/Challenges/FizzBuzz.fs
namespace LeetCode

open BenchmarkDotNet.Attributes
open BenchmarkDotNet.Engines

[<MemoryDiagnoser>]
type Benchmark() =
    let ex1 = LeetCode.Submission.FizzBuzz 3
    let ex2 = LeetCode.Submission.FizzBuzz 5
    let ex3 = LeetCode.Submission.FizzBuzz 15

    [<Benchmark>]
    member _.FizzBuzz() = ex1.Consume

File: Benchmarks.CSharp/Challenges/FizzBuzz.cs
namespace LeetCode;

using BenchmarkDotNet.Attributes;

public partial class Benchmark
{
    [Benchmark]
    public void FizzBuzz()
    {
        Submission.FizzBuzz(3);
        Submission.FizzBuzz(5);
        Submission.FizzBuzz(15);
    }
}
